docs/modules/general/pages/implementation-details/mfe-content-loading/index.adoc

```diff
@@ -0,0 +1,64 @@
+= Microfrontend Content loading in OneCX
+
+== Microfrontend's content loading explained
+When discussing Microfrontend loading mechanisms there are 2 important factors that decide on how loading mechanism work. Those factors are:
+
+* **content type**
+** Module
+** Remote Component
+* **expose method**
+** Angular
+** Webcomponent
+
+All of them are dependent on the Application Microfrontend's content that is meant to be integrated with OneCX platform. Please, make sure to properly configure each integrated Application in Application Store.
+
+IMPORTANT: **It is NOT recommended** to use Angular method. Doing so restraints the platform from using many frameworks with different versions, which means that all Microfrontends would need to use the same technology and version.
+
+NOTE: OneCX platform uses **module-federation** technology integrated with **Webpack** to expose and load Microfrontends. To find out more about how module-federation enables microfrontends, please refer here.
+
+=== Content type
+In the context of content loading, content type **determines which mechanism will be responsible for loading**. Module loading will be handled by the Shell UI router via constructed routing rules and Remote Component loading will be handled by the Slot Component (to which the Remote Component has been assigned).
+
+==== Shell UI router mechanism (Module)
+When dealing with modules, Shell UI is responsible for loading them. As explained in the xref:architecture-overview/shell.adoc[Shell overview], Shell UI first constructs workspace and then handles routing and loading of remotes. Workspace configuration contains all valid routes to Applications registered for a given workspace, each route associated with a Microfrontend's module.
+
+Prior to any routing events, Shell UI initializes router to reflect workspace configuration. Based on the data from backend services about the current workspace, Shell UI creates routing rules to use during the runtime.
+
+Let's take a workspace using user management Application as an example. Each Application in workspace configuration has to have a path defined. This path is used for routing and accessing remote entry file of Application's Microfrontend. Assuming the path is `/user`, Shell UI will create a routing rule specifying that whenever url of the page changes to one starting with `/user` it has to load the Microfrontend's module associated with user management Application.
+
+After routing rules are successfully created, each registered Application should have a Route registered in Shell UI's router. Each Route has a **loadChildren method, which is called whenever route gets activated** and other properties used for route matching.
```

Comment:
whenever the route

Reply:
What do you mean with: "and other properties used for route matching."?

Reply:
The intention here is to highlight that each route has the load children method, but it also has additional properties that are used by the shell during routing. I refactored it a bit so its more clear now maybe? Let me know what you think and maybe its better to remove the properties reference

Reply:
What do you think of the following?

Each Route has a **loadChildren method, which is called whenever route gets activated**, but it also has additional properties that are used by the shell during routing.